website/src/header.rs

```diff
@@ -129,10 +129,15 @@ pub fn HamburgerMenu<G: Html>() -> View<G> {
         ) {
             NavLinks {}
             (if let Some((version, data)) = sidebar.get_clone() {
+                let sidebar_current = SidebarCurrent{
+                    version:version.unwrap_or_else(|| "next".to_string()),
+                    path:"".to_owned(),
+                    data,
+                };
```

Comment:
Nit: fixed formatting. Unfortunately we don't have automatic formatting inside view! macro yet...

Also changed `"".to_owned()` to `String::new()` for consistency with the rest of the codebase.
```suggestion
                let sidebar_current = SidebarCurrent {
                    version: version.unwrap_or_else(|| "next".to_string()),
                    path: String::new(),
                    data,
                };
```